v19.1/bytes.md

```diff
@@ -66,6 +66,41 @@ valid UTF-8 byte sequences; an error is reported otherwise.
 `STRING` values can be cast explicitly to `BYTES`. This conversion
 always succeeds.
 
+### `STRING` vs. `BYTES`
+
+While both `STRING` and `BYTES` can appear to have similar behavior in many situations, one should understand their nuance before casting one into the other.
+
+`STRING` treats all of its data as characters, or more specificially, unicode code points. `BYTES` treats all of its data as a byte string. This difference in implementation can lead to dramatically different behavior. For example, let's take a complex ASCII character such as ☃ ([the snowman emoji](https://emojipedia.org/snowman/)):
```

Comment:
sp: "specifically"

I don't understand, what is the "complex ASCII character ⛄️" This doesn't seem to be ASCII at all but rather Unicode (per https://emojipedia.org/snowman/)?  (Apologies if this was already discussed in engineering review, but I would expect some users to also find this confusing.)